src/main/resources/templates/addBook.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Add Book</title>
+    <link rel="stylesheet" href="../css/reset.css"/>
+    <link rel="stylesheet" href="../css/style.css"/>
+</head>
+<body>
+
+<header>
+    <div id="login">
+        <div id="welcome">
+            <h4>Welcome <span>username</span></h4>
+            <form action="/logout" method="post">
+                <button type="submit">Logout</button>
+            </form>
+        </div>
+        {{#message}}<p id="redAtt">{{.}}</p>{{/message}}
+    </div>
+</header>
+
+
+
+<main class="twoColumn">
+    <section class="leftCol" id="allBooks">
+        <div class="controls">
+            <h1 class="headline">Search books to add to your bookshelf</h1>
+            <form id="searchForm" class="controls">
+                <input id="searchTerm" type="text" placeholder="Search" size="50" name="searchTerm"/>
+                <select id="searchType" name="searchType">
+                    <option selected value="">Keyword</option>
+                    <option value="intitle">Title</option>
+                    <option value="inauthor">Author</option>
+                    <option value="subject">Subject</option>
+                    <option value="isbn">ISBN</option>
+                </select>
+                <button id="searchBtn" type="submit">Search</button>
+            </form>
+        </div>
+        <ul class="bookshelf" id="searchList">
+            <!--<li class="book">-->
```

Comment:
Please start taking out commented code before submission.  I just makes the file look sloppy